src/pivot.coffee

```diff
@@ -584,6 +590,40 @@ callWithJQuery ($) ->
     ###
 
     pivotTableRenderer = (pivotData, opts, rendererType) ->
+        #If we've been asked to generate the totals from a meta-aggregator, walk the data tree here and do that
+        if pivotData.opts.totalsMetaAggregator
```

Comment:
Shouldn't this live in the PivotData class instead of the renderer? It's not a good division of responsibilities to have the renderer do data manipulation.